en-US/firefox/features/sync.lang

```diff
@@ -0,0 +1,81 @@
+## NOTE: url for demo server is: https://www-dev.allizom.org/firefox/sync/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/sync/
+
+
+# HTML page title
+;Use Bookmarks, Tabs and Passwords Across Devices | Firefox
+Use Bookmarks, Tabs and Passwords Across Devices | Firefox
+
+
+# HTML page description
+;Bring your Firefox bookmarks, tabs, or passwords wherever you go with Firefox Sync. Share between computers and mobile devices. Get Firefox now!
+Bring your Firefox bookmarks, tabs, or passwords wherever you go with Firefox Sync. Share between computers and mobile devices. Get Firefox now!
+
+
+;Browse uninterrupted with Firefox Sync
+Browse uninterrupted with Firefox Sync
+
+
+# Displayed only to Firefox user with 31 and later, signed in into Firefox Accounts
+;Sync across devices
+Sync across devices
+
+
+;With Firefox on all your devices, you can access bookmarks, tabs and passwords with one easy sign in.
+With Firefox on all your devices, you can access bookmarks, tabs and passwords with one easy sign in.
+
+
+;Sign in to your Firefox Account on mobile or tablet to get your open tabs, bookmarks and passwords anywhere.
+Sign in to your Firefox Account on mobile or tablet to get your open tabs, bookmarks and passwords anywhere.
+
```

Comment:
sign in = signin or sign-in?

Reply:
That's for en-US to decide though

Reply:
Wait, why should it be "sign-in"? That's the verb, so "Sign in" is correct (like Log in).